examples/Server_With_All_Interfaces-Settings.js

```diff
@@ -19,6 +19,7 @@ var moscaSetting = {
         { type: "https", port: 3001, bundle: true, credentials: { keyPath: SECURE_KEY, certPath: SECURE_CERT } }
     ],
     stats: false,
+    qos2Puback: false, // can set to true if using a client which will eat puback for QOS 2; e.g. mqtt.js
```

Comment:
I'm thinking more about this, and I think this behavior should be a bit more customizable. How about adding it as a string with two options:` 'dropToQoS1', `'ignore'` and `'disconnect'`? Maybe we can call it `onQoS2publish`?